crates/rnote-ui/src/appwindow/actions.rs

```diff
@@ -164,6 +164,21 @@ impl RnAppWindow {
         let action_active_tab_close = gio::SimpleAction::new("active-tab-close", None);
         self.add_action(&action_active_tab_close);
 
+        let color_setters = {
+            let p = self.overlays().colorpicker();
+            [
+                [p.setter_1(), p.setter_2(), p.setter_3()],
+                [p.setter_4(), p.setter_5(), p.setter_6()],
+                [p.setter_7(), p.setter_8(), p.setter_9()],
+            ]
+            .concat()
```

Comment:
Not sure if we really need to split the array for aesthetic purposes, but it does look better ;)

Reply:
I mean I could revert it but… it looks nice so :3

Reply:
let's go with what has the best vibe :)